docs/en/admins/09_AccessControl.md

```diff
@@ -34,6 +34,23 @@ You may alternatively pass a `TRUSTED_PROXY` environment variable in a format co
 
 > ☠️ WARNING: FreshRSS will trust any IP configured in the `trusted_sources` option, if your proxy isn’t properly secured, an attacker could simply attach this header and get admin access.
 
+### Authentik Proxy Provider
+
+If you wish to use external authentication with Authentik, you will need to configure a [Proxy Provider](https://version-2023-10.goauthentik.io/docs/providers/proxy/) with a Property Mapping that tells Authentik to inject the `X-Remote-User` and/or `X-WebAuth-User` headers. You can do so with the following expression:
+
+```python
```

Comment:
That looks more like `javascript` than `python`, doesn't it?

Reply:
They're a heavy Python shop and the tool allows for some embedded Python – these Property Mappings are such a case. I can change to whatever you prefer though. (Or remove per my other comment.)

Reply:
No no, all good then